src/burn/engine/bitsengine.cpp

```diff
@@ -439,11 +442,19 @@ static HRESULT CreateJob(
     hr = pJob->SetNotifyFlags(BG_NOTIFY_JOB_TRANSFERRED | BG_NOTIFY_JOB_ERROR | BG_NOTIFY_JOB_MODIFICATION);
     ExitOnFailure(hr, "Failed to set notification flags for BITS job.");
 
-    hr = pJob->SetNoProgressTimeout(BITSENGINE_NO_PROGRESS_TIMEOUT); // use 2 minutes since default is 14 days.
-    ExitOnFailure(hr, "Failed to set progress timeout.");
+    if (fBackgroundDownload)
+    {
+        hr = pJob->SetPriority(BG_JOB_PRIORITY_LOW);
```

Comment:
Does this mean the Bundle could be running for basically ever (i.e. days)? Big bundle, only downloading in background... hangs around for long time, right?